pkg/proto/datadog/trace/stats.proto

```diff
@@ -72,4 +72,5 @@ message ClientGroupedStats {
 	uint64 topLevelHits = 13; // count of top level spans aggregated in the groupedstats
 	string peer_service = 14; // name of the remote service that the `service` communicated with
 	string span_kind = 15; // value of the span.kind tag on the span
+	repeated string peer_tags = 16; // supplementary tags that further describe peer_service
```

Comment:
> // supplementary tags that further describe peer_service

I don't feel this is descriptive enough. What does "further describe peer_service" mean?

Reply:
+1, will revise